### ISSUE TITLE
Closed tabs show up on panorama view

### ISSUE THREAD

**Issue by spastorino** [projectdelphai/panorama-tab-groups#158]
From time to time closed tabs show up on panorama view and I can't kill them. The way to get rid of them is to move those to a new group and kill the group with all theirs tabs.

**Comment by sleepindawg** [comment]
Have you considered a browser restart as another workaround while waiting for a fix from the devs?

**Comment by spastorino** [comment]
Yes that works too, but in my use case it's better to do what I've described. Restarting the browser means to reload all the tabs.

**Comment by spastorino** [comment]
I think the problem may be that the play doesn't play nice with Temporary Containers plugin.

**Comment by MaxwellDexter** [comment]
I've been getting this issue when tabs rapidly open and close, like when I click on a link from facebook/messenger and it goes through facebook's link system to then open up on the page.
<img width="699" alt="Screen Shot 2020-06-22 at 11 20 47 am" src="https://user-images.githubusercontent.com/10956256/85240229-baba4200-b47a-11ea-8b15-4a7552e573cf.png">
